conans/client/graph/graph_binaries.py

```diff
@@ -343,6 +343,16 @@ def evaluate_graph(self, deps_graph, build_mode, update, remotes, nodes_subset=N
         for node in deps_graph.ordered_iterate(nodes_subset=nodes_subset):
             self._propagate_options(node)
 
+            # Make sure that locked options match
+            if (node.graph_lock_node is not None and
+                    node.graph_lock_node.options is not None and
+                    node.conanfile.options.values != node.graph_lock_node.options):
+                raise ConanException("{}: Locked options do not match computed options\n"
+                                     "Locked options:\n{}\n"
+                                     "Computed options:\n{}".format(node.ref,
+                                                                    node.conanfile.options.values,
+                                                                    node.graph_lock_node.options))
```

Comment:
This should be the other way around, right?

```suggestion
                                                                    node.graph_lock_node.options,
                                                                    node.conanfile.options.values))
```

Reply:
Yep, seems a mistake, fixing it.